### PR TITLE
Update ingest-data-event-hub.md

### PR DESCRIPTION
**Pull request by andresilva-msft**
Include "Mapping name is case sensitive" in note

### CODE REVIEW

**Comment by PRMerger15** [comment]
@andresilva-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Jak-MS** [comment]
@shsagir 

Can you review this PR?

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: data-explorer/ingest-data-event-hub.md [hunk 1]
```diff
@@ -150,6 +150,7 @@ For this article, you use static routing, where you specify the table name, data
     >
     > * You don't have to specify all **Default routing settings**. Partial settings are also accepted.
     > * Only events enqueued after you create the data connection are ingested.
+    > * The Mapping name is case sensitive. Any mismatch might result in no ingested records.
```
Comment (YaelSchuster) [inline data-explorer/ingest-data-event-hub.md:153]:
```suggestion
    > * The mapping name is case-sensitive. A mismatch in mapping name will result in records not being ingested.
```

**Comment by PRMerger15** [comment]
@shsagir : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by shsagir** [comment]
#sign-off

**Comment by PRMerger-2** [comment]
Invalid command: '#sign-off'. Only the assigned author of one or more file in this PR can sign off. @

**Review by shsagir — APPROVED** [review]

**Comment by shsagir** [comment]
#sign-off

**Pull request merged**